frontend/public/co-fetch.js

```diff
@@ -145,10 +145,21 @@ export const coFetchJSON = (url, method = 'GET', options = {}) => {
     if (response.headers.get('Content-Length') === '0') {
       return Promise.resolve({});
     }
+    if (response.headers.get('Content-Type') === 'text/plain') {
+      return response.text();
+    }
     return response.json();
   });
 };
 
+export const coFetchJSON = (url, method = 'GET', options = {}) => {
+  return coFetchCommon(url, method, options, { Accept: 'application/json' });
+};
+
+export const coFetchText = (url, options = {}) => {
+  return coFetchCommon(url, 'GET', options);
```

Comment:
Better to explicitly use `Accept: text/plain`.

Reply:
I didn't see the difference in the two, but I agree, explicit helps future refactors / readability. 👍

Reply:
I didn't see the difference in the two, but I agree, explicit helps future refactors / readability. 👍